src/OpenColorIO/CMakeLists.txt

```diff
@@ -152,12 +152,14 @@ if(NOT WIN32)
 
     # Install the pkg-config file.
 
+    include(GNUInstallDirs)
+
     set(prefix ${CMAKE_INSTALL_PREFIX})
-    set(exec_prefix "\${prefix}")
-    set(libdir "\${exec_prefix}/${CMAKE_INSTALL_LIBDIR}")
-    set(includedir "\${prefix}/${CMAKE_INSTALL_INCLUDEDIR}")
+    set(exec_prefix "${prefix}")
```

Comment:
Preserve the '\' so that the generated pkgconfig file still contains `${exec_prefix}` i.e. it does not expand all the file paths.

Reply:
Thanks @hodoulp, in the end the only change left is just the `include(GNUInstallDirs)`